### PR TITLE
changed outer to 0.06 in dual_panda-interface.l

### PR DESCRIPTION
**Pull request by softyanija**
I changed "outer" in :send-gripper-grasp-action to grasp thicker objects.

### CODE REVIEW

**Comment by ykawamura96** [comment]
@softyanija Thank you for PR!
Could you specify the purpose of this change, also, adding video clip or image also helps.

**Comment by softyanija** [comment]
I wanted to grasp servo and fix to make KXR-repair demo, but couldn't. Arg width was 0 , inner and outer were both 0.005(m) so panda could have objects with a thickness of 5 mm or less. Then I changed outer to 0.06. As a result, panda can grasp servo and other parts of KXR. I'll send clip later.

File: jsk_panda_robot/panda_eus/euslisp/dual_panda-interface.l [hunk 1]
```diff
@@ -260,7 +260,7 @@
                     ))
   ;; gripper action for real-controller
   (:send-gripper-grasp-action
-   (act width speed force &key (wait t) (inner 0.005) (outer 0.005))
+   (act width speed force &key (wait t) (inner 0.005) (outer 0.06))
```
Comment (ykawamura96) [inline jsk_panda_robot/panda_eus/euslisp/dual_panda-interface.l]: Please also make change in :[start-grasp](https://github.com/softyanija/jsk_robot/blob/2d139ea972114e4062184e98f66a40fcc0d619db/jsk_panda_robot/panda_eus/euslisp/dual_panda-interface.l#L159) and[ :stop-grasp](https://github.com/softyanija/jsk_robot/blob/2d139ea972114e4062184e98f66a40fcc0d619db/jsk_panda_robot/panda_eus/euslisp/dual_panda-interface.l#L166) methods e.g. add arguments in the methods to change inner and outer parameters,

**Review by ykawamura96 — CHANGES_REQUESTED** [review]
Please also make change in :start-grasp and :stop-grasp methods e.g. add arguments in the methods to change inner and outer parameters.

**Comment by softyanija** [comment]
I changed :start-grasp and :stop-grasp as the commit message says.

**Comment by softyanija** [comment]
The former GIF is before improvement, the later is after improvement
grasp_failure
grasp_success

**Pull request merged**